spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET #index' do
+    before(:example) { get '/users/2/posts' } # get(:index)
+    it 'it is a success' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it "it renders 'index' template" do
+      expect(response).to render_template('index')
+    end
+
```

Comment:
- [ ] Tests relating to this `render_template` are failing. Because you have not installed the `rails-controller-testing` gem. You need to add the gem to your test environment in the Gemfile and run bundle install for the tests to pass.
  ![temp](https://user-images.githubusercontent.com/81468675/186172166-4d61c8e8-fca3-4ed9-9cc8-5afe463591d1.jpg)
  ![rails-g](https://user-images.githubusercontent.com/81468675/186172173-241bba2d-a69d-4f01-8bde-ca29eabb5b14.jpg)